src/routes/docs/tutorials/apple/step-8/+page.markdoc

```diff
@@ -0,0 +1,77 @@
+---
+layout: tutorial
```

Comment:
I would consider removing this for the sake of brevity. This is cool but probably out of scope.

WDYT?

Reply:
No p